docs/docs/end-to-end-testing.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "End-to-end testing"
+---
+
+[Cypress](https://www.cypress.io/) is one of the options when it comes to end-to-end testing. Setting up and writing tests in Cypress is developer-friendly and easy to learn. You'll profit from the time travel feature or the extensive debugging capabilities with Chrome DevTools. Of course you can also use it with Gatsby and this guide will explain how.
```

Comment:
Try and avoid "easy to learn" since it could be hard for some people. Can you be more specific about why you think it's easy and developer-friendly and better than other options?
For example, maybe it takes fewer steps to install it than other options, it ingrates into a workflow, etc.

Reply:
I actually wrote that on purpose as I honestly think that it’s easy to learn and therefore is an essential feature to this library. I‘ll try to expand the why’s